src/main/java/step2/domain/Lotto.java

```diff
@@ -5,32 +5,30 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 import java.util.stream.Collectors;
 
 public class Lotto {
-    private List<Integer> lottoNumber;
-    private WinningPrice winningPrice;
+    private List<LottoNumber> lottoNumber;
 
-    public Lotto(List<Integer> lottoNumber, WinningPrice winningPrice) {
+    public Lotto(List<LottoNumber> lottoNumber) {
         this.lottoNumber = lottoNumber;
-        this.winningPrice = winningPrice;
     }
 
-    public Lotto(List<Integer> lottoNumber) {
-        this(lottoNumber, WinningPrice.OTHER);
-    }
-
-    public List<Integer> getLottoNumber() {
+    public List<LottoNumber> getLottoNumber() {
         return lottoNumber;
```

Comment:
Collections.unmodifiablelist()를 활용하여 read-only로 리턴하여 외부에서 컬렉션의 변조를 막도록 한다면 좋을 것 같아요. :)

```suggestion
        return Collections.unmodifiablelist(lottoNumber);
```

Reply:
으어..감사합니다..또한번 배웁니다.ㅠㅠㅠ